packages/fuel-gauge/test/fixtures/forc-projects/Forc.toml

```diff
@@ -58,3 +58,6 @@ members = [
   "vector-types-script",
   "vectors",
 ]
+
+[patch.'https://github.com/fuellabs/sway']
+std = { git = "https://github.com/fuellabs/sway", branch = "xunilrj/abi-decode-vec-with-capacity" }
```

Comment:
Reminder to remove the patch branch when we have a new `forc` release containing fixes.